plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -1265,11 +1266,11 @@ private ExecutionResult prepareNetworkElementCommand(IpAssocCommand cmd) {
 
     @Override
     public ExecutionResult executeInVR(String routerIP, String script, String args) {
-        return executeInVR(routerIP, script, args, 120);
+        return executeInVR(routerIP, script, args, Duration.standardSeconds(120L));
```

Comment:
Please consider extracting ``Duration.standardSeconds(120L)`` to a constant to clarify intent